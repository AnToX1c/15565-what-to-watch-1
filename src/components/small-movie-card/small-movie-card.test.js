import React from "react";
import renderer from "react-test-renderer";
import SmallMovieCard from "./small-movie-card";

const mock = {
  id: 1,
  title: `Fantastic Beasts: The Crimes of Grindelwald`,
  imgsrc: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`
};

it(`SmallMovieCard page correctly renders`, () => {
  const tree = renderer
    .create(<SmallMovieCard film={mock} onMouseOver={jest.fn()}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
