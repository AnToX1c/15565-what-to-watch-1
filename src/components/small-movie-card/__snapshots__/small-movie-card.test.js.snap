// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallMovieCard page correctly renders 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <video
    height="175"
    loop={true}
    muted={true}
    poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
    width="280"
  >
    <source
      src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
    />
  </video>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Fantastic Beasts: The Crimes of Grindelwald
    </a>
  </h3>
</article>
`;
